### PR TITLE
Hotfix mlm

### PR DESCRIPTION
**Pull request by petrsokerin**
fixed bug with mlm

### CODE REVIEW

File: config.ini [hunk 1]
```diff
@@ -25,9 +25,9 @@ period                  = day
 
 [Tr2Vec]
 window_size     = 10
-mcc_vocab_size  = 344
+mcc_vocab_size  = 345
 mcc_emb_size    = 16
-amnt_bins       = 50
+amnt_bins       = 51
```
Comment (nokiroki) [inline config.ini:30]: Why we need 51 bins when we are masing only mccs?

File: models/mask_language_model/base_model.py [hunk 2]
```diff
@@ -54,23 +54,23 @@ def _mean_pooling(self, outputs: torch.Tensor, lengths: torch.Tensor) -> torch.T
 
     def training_step(self, batch: torch.Tensor, batch_idx: int) -> STEP_OUTPUT:
         mcc_seqs, amnt_seqs, _, lengths, avg_amnt, top_mcc = batch
-
+        labels = mcc_seqs.clone().view(-1)
 
         new_batches, rand_masks = masking_all_batches(
             list_changing_batches=[mcc_seqs, amnt_seqs], 
-            mask_tokens = [self.mcc_vocab_size + 1, self.mcc_amnt_seqs + 1],
```
Comment (nokiroki) [inline models/mask_language_model/base_model.py:61]: + 1 was responsible for padding token. Is it really necessary to remove it?

File: models/mask_language_model/base_model.py [hunk 3]
```diff
@@ -95,10 +95,11 @@ def training_epoch_end(self, outputs: torch.Tensor) -> Optional[STEP_OUTPUT]:
 
     def validation_step(self, batch: torch.Tensor, batch_idx: int) -> Optional[STEP_OUTPUT]:
         mcc_seqs, amnt_seqs, labels, lengths, avg_amnt, top_mcc = batch
+        labels = mcc_seqs.clone().view(-1)
 
         new_batches, rand_masks = masking_all_batches(
             list_changing_batches=[mcc_seqs, amnt_seqs], 
-            mask_tokens = [self.mcc_vocab_size + 1, self.mcc_amnt_seqs + 1],
```
Comment (nokiroki) [inline models/mask_language_model/base_model.py:101]: Same

File: models/mask_language_model/base_model.py [hunk 4]
```diff
@@ -128,10 +129,12 @@ def validation_epoch_end(self, outputs: Union[EPOCH_OUTPUT, List[EPOCH_OUTPUT]])
 
     def test_step(self, batch: torch.Tensor, batch_idx: int) -> Optional[STEP_OUTPUT]:
         mcc_seqs, amnt_seqs, labels, lengths, avg_amnt, top_mcc = batch
+        labels = mcc_seqs.clone().view(-1)
+
 
         new_batches, rand_masks = masking_all_batches(
             list_changing_batches=[mcc_seqs, amnt_seqs], 
-            mask_tokens = [self.mcc_vocab_size + 1, self.mcc_amnt_seqs + 1],
```
Comment (nokiroki) [inline models/mask_language_model/base_model.py:134]: Same

File: models/mask_language_model/transaction_gru.py [hunk 5]
```diff
@@ -30,7 +30,7 @@ def __init__(
         *args: Any,
         **kwargs: Any
     ) -> None:
-        super().__init__(output_dim, *args, **kwargs)
+        super().__init__(output_dim, output_dim, mcc_vocab_size, amnt_bins, *args, **kwargs)
```
Comment (nokiroki) [inline models/mask_language_model/transaction_gru.py]: Two output dims?

File: utils/data_utils/utils.py [hunk 6]
```diff
@@ -129,10 +132,10 @@ def masking_all_batches(
 
     for idx_batch, batch in enumerate(list_changing_batches):
 
-        new_batch, mask_for_batch = masking_one_batch(batch, rand_mask, mask_tokens[idx_batch], save_tokens[idx_batch])
-
-        rand_masks_per_batch.append(mask_for_batch)
+        new_batch, mask_for_batch = masking_one_batch(batch, rand_mask, mask_tokens[idx_batch],
```
Comment (nokiroki) [inline utils/data_utils/utils.py]: Appropriate tabs

**Review by nokiroki — CHANGES_REQUESTED** [review]

File: config.ini (hunk 1 above)
Comment (petrsokerin) [reply config.ini:30]: I masked amount too

File: models/mask_language_model/transaction_gru.py (hunk 5 above)
Comment (petrsokerin) [reply models/mask_language_model/transaction_gru.py]: cosyak

File: config.ini (hunk 1 above)
Comment (nokiroki) [reply config.ini:30]: Yeah, I am stupid)))

File: utils/data_utils/utils.py (hunk 6 above)
Comment (nokiroki) [reply utils/data_utils/utils.py]: Fine, I will do it myself

**Review by nokiroki — APPROVED** [review]

**Pull request merged**